int64 numLego
int64 lego1_imgx
int64 lego1_imgy
float64 lego1_x
float64 lego1_y
float64 lego1_q
float64 lego1_w
float64 lego1_h
int64 lego1_p
int64 lego2_imgx
int64 lego2_imgy
float64 lego2_x
float64 lego2_y
float64 lego2_q
float64 lego2_w
float64 lego2_h
int64 lego2_p
int64 lego3_imgx
int64 lego3_imgy
float64 lego3_x
float64 lego3_y
float64 lego3_q
float64 lego3_w
float64 lego3_h
int64 lego3_p